### PR TITLE
mgr/dashboard: Confirmation modal doesn't close

### PR DESCRIPTION
**Pull request by votdev**
Fixes: https://tracker.ceph.com/issues/24729

Signed-off-by: Volker Theile <vtheile@suse.com>

- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [x] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by tspmelo** [comment]
jenkins retest this please

File: src/pybind/mgr/dashboard/frontend/src/app/shared/components/confirmation-modal/confirmation-modal.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<cd-modal>
+<cd-modal (hide)="cancel($event)">
```
Comment (s0nea) [inline src/pybind/mgr/dashboard/frontend/src/app/shared/components/confirmation-modal/confirmation-modal.component.html]: Make check is issuing the following: 

```ERROR in src/app/shared/components/confirmation-modal/confirmation-modal.component.html(1,11): : Expected 0 arguments, but got 1.```

**Comment by callithea** [comment]
jenkins test make check

File: src/pybind/mgr/dashboard/frontend/src/app/shared/components/confirmation-modal/confirmation-modal.component.html (hunk 1 above)
Comment (votdev) [reply src/pybind/mgr/dashboard/frontend/src/app/shared/components/confirmation-modal/confirmation-modal.component.html]: Fixed

**Pull request merged**